scripts/publish-process/util.js

```diff
@@ -429,7 +456,7 @@ const performPublish = async (changes, changeSummary) => {
   if (!testing) {
     disableKrypton();
     execSync(`git add . && git commit -m "${commitMessage}" --no-verify`);
-    await publishPackagesToNPM(changes);
+    await publishPackagesToNPM(changes, null);
```

Comment:
so you don't need to pass null here you could modify the function declaration to be `tag = null` for the default case